### PR TITLE
Flashing ODROID: add info that Allen key is needed

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- as not everyone has one of these at home
- minor editing: turn running text into numbered steps.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/common-tasks/flashing_n2_otg.md [hunk 1]
```diff
@@ -60,10 +61,19 @@ This will configure the ODROID-N2+ and OTG to act as a memory card reader:
...
+1. Connect the ODROID-N2+ to your PC via the micro-USB port at the front of the ODROID-N2+. 
+1. When the ODROID-N2 is recognized as a USB connected storage device, you can flash the eMMC with [Etcher](https://www.balena.io/etcher/).
+   * Use the latest stable version of Home Assistant OS for the [ODROID-N2+](https://github.com/home-assistant/operating-system/releases/download/{{site.data.version_data.hassos['odroid-n2']}}/haos_odroid-n2-{{site.data.version_data.hassos['odroid-n2']}}.img.xz) (haos_odroid-n2-{{site.data.version_data.hassos['odroid-n2']}}.img.xz).
 
-When the flash process is complete, disconnect the ODROID-N2+ from your PC and remove the power cable. Remove the USB and HDMI cable, and make sure to toggle the boot mode switch back to MMC.
+1. When the flash process is complete, disconnect the ODROID-N2+ from your PC.
+   * Remove the power cable.
+   * Remove the USB and HDMI cable.
```
Comment (frenck) [inline source/_includes/common-tasks/flashing_n2_otg.md]:
```suggestion
   * Remove the USB and HDMI cables.
```

File: source/_includes/common-tasks/flashing_n2_otg.md [hunk 2]
```diff
@@ -60,10 +61,19 @@ This will configure the ODROID-N2+ and OTG to act as a memory card reader:
...
+   * Make sure to toggle the boot mode switch back to MMC.
 
-Once it is back in its case, connect your ODROID-N2+ to your network with an Ethernet cable and plug in power.
+1. Put the ODROID back in its case.
+1. Connect your ODROID-N2+ to your network with an Ethernet cable and plug in power.
 
-If your router supports mDNS, you will be able to reach your installation on `http://homeassistant.local:8123`.  If your network doesn’t support mDNS, you’ll have to use the IP address of your ODROID-N2+ instead of `homeassistant.local`. For example, `http://192.168.0.9:8123`. You should be able to find the IP address of your ODROID-N2+ from the admin interface of your router.
+1. If your router supports mDNS, you will be able to reach your installation on `http://homeassistant.local:8123`.
```
Comment (frenck) [inline source/_includes/common-tasks/flashing_n2_otg.md]:
```suggestion
1. If your router supports mDNS, you can reach your installation at `http://homeassistant.local:8123`. 
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by c0ffeeca7** [comment]
thank you for the catch, @frenck!

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**